src/components/VotedRepoListWrap.tsx

```diff
@@ -40,19 +40,36 @@ const VotedRepoListWrap = (): JSX.Element => {
     setSearchParams({ limit: String(limit + 10) });
   };
 
+  if (!isLoading && !data.length) {
+    return (
+      <div className="mx-auto max-w-7xl px-4 pb-1 mt-10">
+        <div className="flex flex-col gap-y-5 overflow-hidden mb-12">
+          <div className="flex items-center gap-x-2.5">
+            <BsFillCalendar2Fill className="w-8 h-8 text-white" />
+
+            <h1 className="text-2xl text-white font-semibold">
+              My Votes Repositories
+            </h1>
+          </div>
+
+          <div
+            className="p-4 border rounded-2xl bg-white w-full space-y-1 relative"
+          >
+            <h1 className="text-2xl text-center">You didn&apos;t vote on any Repositories. Your votes will appear here.</h1>
```

Comment:
Not quite confident with this copy. My suggestion is a bit nicer for the viewers
```suggestion
            <h1 className="text-2xl text-center">Try voting on any repository. Your vote will appear here.</h1>
```